### ISSUE TITLE
Low Level Function / Constant inconsistency

### ISSUE THREAD

**Issue by Lokathor** [HiTECNOLOGYs/cl-charms#19]
The charms/ll package uses cffi:translate-name-from-foreign name as it defines the FFI functions, but it does not use it when it defines FFI constants. It should probably use the same translation on both for consistency.

**Comment by tarballs-are-good** [comment]
My own preference for constants from the C world is to write them in Lisp style but prepend them with the `$` sign, e.g., `WA_ALTCHARSET` would be `$WA-ALTCHARSET`. I am hesitant to have a bunch of 1-to-1 abbreviated, lowercase constant names without some indicator that they are constant.

**Comment by Lokathor** [comment]
Yeah, converting them all to Lisp style naming with - instead of _ would be best.

I'm not sure if a prefix character is needed necessarily though. That's feels like a "if you can't read enough of what's going on ahead of time to know what's a constant and what's a function maybe don't use the low-level section" sort of deal.
